django/db/backends/postgresql/features.py

```diff
@@ -55,6 +55,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_aggregate_filter_clause = True
     supported_explain_formats = {'JSON', 'TEXT', 'XML', 'YAML'}
     validates_explain_options = False  # A query will error on invalid options.
+    has_native_serial_field = True
```

Comment:
Please position this up near `has_native_uuid_field` and `has_native_duration_field`.

You also need to add `has_native_serial_field = False` to `django/db/backends/base/features.py`.